rpi_backlight/__init__.py

```diff
@@ -24,7 +24,7 @@ class BoardType(Enum):
     #: Tinker Board 2
     TINKER_BOARD_2 = 3
     #: Microsoft Surface RT
-    MICROSOFT_SURFACE_RT = 4
+    SURFACE_RT_OR_GOOGLE_CHROMEBOOK = 4
```

Comment:
Please call it GENERIC at this point

Reply:
The problem with generic is that, most generic devices have brightness control at `sys/class/backlight/acpi_video0` or `sys/class/backlight/intel_backlight` (and those devices would be supported by i386 version of Raspberry Pi OS). Given that, what do you think we should do?

Reply:
This is a library intended to run on a raspberry pi board with the official raspberry pi display, edge cases like this don't matter